packages/blocks/src/__internal__/utils/index.ts

```diff
@@ -14,5 +14,6 @@ export * from './lit.js';
 export * from './query.js';
 export * from './rect.js';
 export * from './selection.js';
+export * from './shortcut.js';
```

Comment:
According to CI log, this is better not be exposed at top level since it breaks SSR by accessing `navigator`.